app/src/main/java/com/mmm/parq/activities/DriverActivity.java

```diff
@@ -248,16 +248,17 @@ public void onSaveInstanceState(Bundle savedInstanceState) {
 
     // Implementing OnNavigationCompletedListener Interface
     @Override
-    public void showEndReservationFragment(double cost) {
-            DriverEndReservationFragment driverEndReservationFragment = new DriverEndReservationFragment();
+    public void showReviewFragment() {
+            DriverReviewFragment driverReviewFragment = new DriverReviewFragment();
             FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
+
             // TODO(kenzshelley) Remove this once Reservations include cost themselves.
```

Comment:
random and unrelated to this PR: how would we include cost on the back end? Would we just calculate it every time they GET? What if we added a getCost() to the model on Android